net-mgmt/pfSense-pkg-arpwatch/files/usr/local/pkg/arpwatch.inc

```diff
@@ -61,7 +62,7 @@ function arpwatch_install_command() {
 
 			touch($arp_file);
 
-			$rc['start'] .= "/usr/local/sbin/arpwatch $disable_carp $disable_bogons $disable_zero -f $arp_file -i $ifname -m $notifications_recipient"."\n";
+			$rc['start'] .= "/usr/local/sbin/arpwatch $disable_carp $disable_bogons $disable_zero $enable_zeropad -f $arp_file -i $ifname -w $notifications_recipient"."\n";
```

Comment:
As long as you're making changes here, `$arp_file`, `$ifname`, and `$notifications_recipient` should not be used directly but should be passed through `escapeshellarg()` to protect the full command against malformed user data. Since the other variables are explicitly set to predefined strings, they are OK as-is.

Reply:
I don't quite understand how to update net-mgmt/arpwatch. Should I call out a specific svn revision from the FreeBSD ports tree, e.g. r520959?

I've added the escapeshellarg() calls (and tried to make the string formatting a bit more readable).

Reply:
>  don't quite understand how to update net-mgmt/arpwatch. Should I call out a specific svn revision from the FreeBSD ports tree, e.g. r520959?

That's a start, assuming it was just one commit that did it. A git commit id would be better (You can find it on the master branch here, or https://github.com/freebsd/freebsd-ports/ )

Reply:
@leres I take care of cherry-picking net-mgmt/arpwatch commits from upstream. Please re-submit just changing pfSense-pkg-arpwatch and add a note that version 3.0 is required.